test/server.test.ts

```diff
@@ -105,4 +107,47 @@ describe("server", () => {
       });
     });
   });
+
+  describe("customer", () => {
+    const indicesOfSeedCust = [...Array(seedCollCust.length).keys()];
+    indicesOfSeedCust.forEach((index) => {
+      const expectedCust = seedCollCust[index];
+      const expectedUsername = expectedCust.username;
```

Comment:
`npm test` did execute this test case for all customers added to `seed.ts` (`username: ['appleseed', 'poppins']`):
```shell
 PASS  test/server.test.ts
  server
    transactionBatch
      ✓ returns the correct transaction batch for the given accountId: 123456 (24 ms)
      ✓ returns the correct transaction batch for the given accountId: 333333 (1 ms)
    customer
      ✓ returns the correct customer for the given username: appleseed (2 ms)
      ✓ returns the correct customer for the given username: poppins
```